src/database/parse-server.ts

```diff
@@ -0,0 +1,48 @@
+import { DatabaseConfig } from "./database-config";
+
+export default class ParseServer implements Database {
+  private static instance: Database;
+
+  private Parse: any;
+  private Event: any;
+
+  public static getInstance(Parse: any, AsyncStorage: any): Database {
```

Comment:
nit: why isnt this func in the database interface?

Reply:
The `getInstance()` method is pretty specific to Singletons, so I didn't want to require the method. Also it takes in some pretty specific parameters for Parse. I think it's probably not necessary for the interface.